frontend/lib/resolution/resolution-queries.cpp

```diff
@@ -3593,7 +3531,7 @@ gatherAndFilterCandidates(Context* context,
                                   rejected);
 
     // filter out last resort candidates
-    CandidatesVec lrcGroup;
+    CandidatesAndForwardingInfo lrcGroup;
```

Comment:
Just noting that forwarding info was not used in these sections of the code prior, but a small amount of space is allocated for it here. I think it's fine, but there may be performance implications. Consider noting in a comment?

If it becomes a problem, I suspect it might be nice to allow `CandidatesVec` to be used without `CandidatesAndForwardingInfo`, and add methods to the latter that can accept the former. Not necessary right now IMO, though.

Reply:
The additional memory used would just be the empty `forwardingInfo` vector right?
Would you want that commented in `CandidatesAndForwardingInfo`, or in places where it's used without forwarding info?

Reply:
Either place is fine, thanks. And yes, it's just the overhead for an empty vector -- which I think is something around 2 words?